vector/src/main/java/im/vector/app/features/spaces/leave/SpaceLeaveAdvancedViewModel.kt

```diff
@@ -72,14 +72,14 @@ class SpaceLeaveAdvancedViewModel @AssistedInject constructor(
                     try {
                         state.selectedRooms.forEach {
                             try {
-                                session.getRoom(it)?.leave(null)
+                                session.leaveRoom(it)
```

Comment:
Behind the scene I think this is equivalent to leave a room or a space, but can `it` be a `spaceId` sometimes here?

Reply:
As far i understand we leave all selected rooms in a space here. Maybe there could be also subspaces, but I think rooms is more often and more relevant here.